pkg/pillar/persistcache/README.md

```diff
@@ -0,0 +1,36 @@
+# Persisted Cache (persistcache)
+
+## Use-case
+
+This package implements a persistance cache. The main use-case is to store data on filesystem and
+provide access to it via API. Limitation of current implementation is that data should be represented
+as string. This was developed for configuration services feature in EVE. Since there is an option for 
```

Comment:
string implies valid UTF-8. Can't you define it to be a byte slice instead? That is much more general.

Reply:
Makes sense I changed it to byte slice